README.md

```diff
@@ -216,6 +402,32 @@ void parallelize(int count) {
 }
 ```
 
-Since the _Overall/RunScripts_ permission is required to push any changes to the global library repository,
-library code is allowed to call any Java methods without sandbox checks.
-Therefore it can safely encapsulate calls to third-party libraries (as well as to Jenkins model APIs, etc.).
+Libraries are cached by default in `~/.groovy/grapes/` on the Jenkins master.
```

Comment:
Well `System.getProperty("user.home")` is going to return some directory, I do not offhand know what. Depends on the installation package.